public/js/webcam.js

```diff
@@ -66,26 +76,22 @@ function postTracks(dataURL) {
         });
 }
 
-getTracksButton.addEventListener("click", ()=>{
-    console.log("CLICK");
-    const dataURL = canvas.toDataURL();
-    webcam.style.display = "none";
-    loadingDiv.style.display = "flex";
-    postTracks(dataURL);
+getTracksButton.addEventListener("click", () => {
+    const dataURL = canvas.toDataURL('image/png', 1);
```

Comment:
What does the 1 do here?

Reply:
Oh i think its the quality of the picture, so 1 retains the best quality